src/main/resources/changesets/changelog-consent-54.0.xml

```diff
@@ -0,0 +1,35 @@
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet dbms="postgresql" author="grushton" id="54.0">
+        <sql>
+            UPDATE data_access_request
+            SET data = replace(data::TEXT,'\"ontologies\":[null]','\"ontologies\":[]')::jsonb
+            WHERE (data #>> '{}')::jsonb->>'ontologies' = '[null]';
+        </sql>
+        <sql>
+            UPDATE data_access_request
+            SET data = replace(data::TEXT,'\"ontologies\":[{}]','\"ontologies\":[]')::jsonb
+            WHERE (data #>> '{}')::jsonb->>'ontologies' = '[{}]';
+        </sql>
+        <sql>
+            UPDATE data_access_request
+            SET data = replace(data::TEXT,'\"ontologies\":[{},{}]','\"ontologies\":[]')::jsonb
+            WHERE (data #>> '{}')::jsonb->>'ontologies' = '[{},{}]';
+        </sql>
+        <sql>
+            UPDATE data_access_request
+            SET data = replace(data::TEXT,'\"ontologies\":[{},{},{}]','\"ontologies\":[]')::jsonb
+            WHERE (data #>> '{}')::jsonb->>'ontologies' = '[{},{},{}]';
+        </sql>
+        <sql>
+            UPDATE data_access_request
+            SET data = replace(data::TEXT,'\"ontologies\":[{},{},{},{}]','\"ontologies\":[]')::jsonb
```

Comment:
Do we know that the corrupt data is at most 4 empty objects for ontologies on prod?

Reply:
No, I've only seen up to 3 in prod, but this is just an extra check in case we have these on dev or staging.